include/glob.hpp

```diff
@@ -28,47 +29,82 @@ namespace glob {
 
 		/* Add */
 		vect operator+(const vect &v) const { return vect(x + v.x,
-		                                                  y + v.y); };
+														  y + v.y); };
 
 		/* Subtract */
 		vect operator-(const vect &v) const { return vect(x - v.x,
-		                                                  y - v.y); };
+														  y - v.y); };
 
 		/* Multiply by a scalar */
 		vect operator*(const float &s) const { return vect(x * s,
-		                                                   y * s); };
+														   y * s); };
 
 		/* Divide by a scalar */
 		vect operator/(const float &s) const { return vect(x / s,
-		                                                   y / s); };
+														   y / s); };
```

Comment:
The above spaces were used for alignment. You can not use tabs as aligning characters because not everyone uses a tab width of 4. The problem is demonstrated above. GitHub uses a tab width of 1, so they are not aligned.